packages/gallery/src/components/item/itemView.js

```diff
@@ -595,6 +596,7 @@ class ItemView extends GalleryComponent {
     const itemDoesntHaveLink = !this.itemHasLink(); //when itemClick is 'link' but no link was added to this specific item
 
     const itemStyles = {
+      transition: ANIMATIONS[styleParams.imagePlacementAnimation],
```

Comment:
constant is not code.
'all 1s ease' is not a constant.
tansition: styleParams.imagePlacementAnimation === ANIMATIONS.SLIDE ? 'all 1s ease' : 'none'